standard/classes.md

```diff
@@ -722,15 +722,15 @@ class_body
     ;
 ```
 
-### 15.2.7 Partial declarations
+### 15.2.7 Partial type declarations
 
-The modifier `partial` is used when defining a class, struct, or interface type in multiple parts. The `partial` modifier is a contextual keyword ([§6.4.4](lexical-structure.md#644-keywords)) and only has special meaning immediately before one of the keywords `class`, `struct`, or `interface`.
+The modifier `partial` is used when defining a class, struct, or interface type in multiple parts. The `partial` modifier is a contextual keyword ([§6.4.4](lexical-structure.md#644-keywords)), which in this context has special meaning immediately before the keywords `class`, `struct`, and `interface`.
```

Comment:
It's also used when defining a partial method... Hmm.

Reply:
Maybe a note and a cross-reference to the section on partial members?

Reply:
@RexJaeschke to have another go at this paragraph.

Reply:
OK; I changed the para, and added a reference to partial methods.